docs/administration/how_to_guides/organization_management/data_purging_compliance.mdx

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 10
+---
+
+# Data Purging for Compliance
+
+This guide covers the various features available after data reaches LangSmith Cloud servers to help you achieve your privacy goals.
+
+## Data Retention
+
+LangSmith provides automatic data retention capabilities to help with compliance and storage management. Data retention policies can be configured at the organization and project levels.
+
+For detailed information about data retention configuration and management, please refer to the [Data Retention concepts](/administration/concepts#data-retention) documentation.
+
+## Trace Deletes
+
+Trace deletes can be done self-serve via our API. The API supports two methods for deleting traces:
+
+1. **By trace IDs and session ID**: Delete specific traces by providing a list of trace IDs and their corresponding session ID (up to 1000 traces per request)
+2. **By metadata**: Delete traces across a workspace that match any of the specified metadata key-value pairs
+
+Link To API Spec: https://api.smith.langchain.com/redoc#tag/run/operation/delete_runs_api_v1_runs_delete_post
+
+:::warning Trace Deletes
+All trace deletions will delete related entities like feedbacks, aggregations, and stats across all data storages.
+:::
+
+### Deletion Timeline
+
+Trace deletions are processed during non-peak usage times and are not instant, usually within a few hours. There is no confirmation of deletion - you'll need to query the data again to verify it has been removed.
+
+### Delete by Metadata
+
+When deleting by metadata:
+
+- Accepts a `metadata` object of key/value pairs. KV pair matching uses an OR condition. A trace will match if it has ANY of the key-value pairs specified in metadata (not all)
+- You don't need to specify a session id when deleting by metadata. Deletes will apply across the workspace.
+
+## Sample Trace Delete Requests
```

Comment:
Just a suggestion, so feel free to leave: 

I would possibly move the two example commands to within the section e.g., Delete by metadata example goes under the Delete by metadata section. Then, we're not duplicating that heading and it consolidates the content by removing the need for the ## Sample Trace section.

(Same for ### Delete specific traces as well)